x/emissions/proto/emissions/v1/genesis.proto

```diff
@@ -135,16 +137,21 @@ message GenesisState {
   // all network inferer (3rd index) regrets L_ij made under the regime of the one-in forecaster (2nd index)
   repeated TopicIdActorIdActorIdTimeStampedValue latestOneInForecasterNetworkRegrets = 47;
   // the forecaster (2nd index) regrets made under the regime of the same forecaster as a one-in forecaster
-  repeated TopicIdActorIdTimeStampedValue latestOneInForecasterSelfNetworkRegrets = 48;
+  repeated TopicIdActorIdTimeStampedValue latestNaiveInfererNetworkRegrets = 48;
```

Comment:
i hope it's ok that `latestOneInForecasterSelfNetworkRegrets` -> `latestNaiveInfererNetworkRegrets` share the same number


but maybe this is a problem we can look at when we look at the upgrade, Fernando has more experience